src/admin/participants/handler.js

```diff
@@ -138,12 +162,43 @@ const addLimitAndInitialPosition = async function (request, h) {
   }
 }
 
+const getLimits = async function (request, h) {
+  Sidecar.logRequest(request)
+  try {
+    const result = await Participant.getLimits(request.params.name, request.query)
+    let limits = []
+    if (Array.isArray(result) && result.length > 0) {
+      result.forEach(item => {
+        limits.push({
+          currency: (item.currencyId || request.query.currency),
+          type: item.name,
+          value: item.value
+        })
+      })
+    }
+    return limits
+  } catch (err) {
+    throw Boom.badRequest()
+  }
+}
+
+const adjustLimits = async function (request, h) {
+  Sidecar.logRequest(request)
+  try {
+    await Participant.adjustLimits(request.params.name, request.payload)
```

Comment:
Are we only catering for a single limit and not a list of limits?

I feel there is inconsistency here between the GET and the POST. Where the GET returns a list and the POST expects a single limit.

This does break the REST design paradigm.

Reply:
Modified the GET to match with POST object
For now we are not considering the payload to be an array of multiple limits for POST, @NicoDuvenage  will create a separate story for that.

Reply:
Changes/Updates
1. changed the output of GET
2. changed the output of POST
3.  updated the sequence diagram for POST
4. created the sequence diagram for GET
5. raised PR for these 2 diagrams "https://github.com/mojaloop/central-ledger/pull/103"
6. added few more coverage tests
7. created the story #383 to cater to multiple limits in POST